packages/components/src/internal/components/labels/APIWrapper.ts

```diff
@@ -21,46 +21,19 @@ function handleBarTenderConfigurationResponse(response: any): BarTenderConfigura
     return new BarTenderConfiguration(btConfiguration);
 }
 
-function createLabelTemplateList(): Promise<DomainDesign> {
+function createLabelTemplateList(): Promise<LabelTemplate[]> {
```

Comment:
I would probably have this be a `Promise<void>` since it always returns an empty array, but this works and is consistent with the expected result for the caller. 

If the list already exists, the ensure API call returns nothing, but will the caller of this method get those existing template some other way?